migrations/src/9_deploy_razor_and_faucet.js

```diff
@@ -0,0 +1,33 @@
+const { NETWORK, RAZOR_ADDRESS } = process.env;
+const {
+  deployContract,
+  appendDeploymentFile,
+  readOldDeploymentFile,
+} = require('../migrationHelpers');
+
+const { BigNumber } = ethers;
+const initialSupply = (BigNumber.from(10).pow(BigNumber.from(27)));
+const deployRAZORAndFaucet = async () => {
+  if (NETWORK !== 'mainnet' && RAZOR_ADDRESS === '') {
+    const RAZOR = await deployContract('RAZOR', [], [initialSupply]);
+    await deployContract('Faucet', [], [RAZOR.address]);
+  } else {
+    const { Faucet, RAZOR } = await readOldDeploymentFile();
```

Comment:
Faucet will not be deployed on mainnet FYI

Reply:
ok

Reply:
@hrishikeshio At line number 11 we are checking if its mainnet or not, if its not mainnet faucet wont be deployed.